packages/clerk-js/src/ui/localization/makeLocalizable.tsx

```diff
@@ -89,15 +89,17 @@ export const useLocalizations = () => {
     }
 
     const { code, message, longMessage, meta } = (error || {}) as ClerkAPIError;
-    const { paramName = '' } = meta || {};
+    const { paramName = '', identifiers, emailAddresses } = meta || {};
```

Comment:
Are these concatenated if returned as arrays?

Reply:
Similarly, per the example, `not_allowed_access: '{{identifiers}} is not allowed to access this page!!!!!'` - would this be confusing there are multiple `identifiers` (or `emailsAddresses`) returned.

Reply:
These are always arrays with 1 or more items.

Yes, the error message needs to be agnostic on the "is"/"are" part.